include/zephyr/sys/shutdown.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2015 Wind River Systems, Inc.
```

Comment:
Incorrect copyright notice, here and in `shutdown_internal.h`.

Reply:
fixed